timescaledb/getting-started/compress-data.md

```diff
@@ -11,13 +11,13 @@ All postgresql data types can be used in compression.
 
 At a high level, TimescaleDB's built-in job scheduler framework will asynchronously convert recent data from an uncompressed row-based form to a compressed columnar form across chunks of TimescaleDB hypertables.
 
-Let’s set up a compression policy on our hypertable to see how it works.
+Let's set up a compression policy on our hypertable to see how it works.
```

Comment:
```suggestion
Let's enable compression on our hypertable and then look at two ways of compressing data: with an automatic policy or manually.
```
(I'm not beholden to this wording, but I've seen confusion over enabling compression on the table vs. actually compressing the chunks. Enabling it (`ALTER TABLE`) does not compress anything in and of itself.)